app/views/user_mailer/batch_subscription_notification.text.erb

```diff
@@ -3,29 +3,44 @@
 <% @seen_summary = {} %>
 <% @creations.each_with_index do |creation, index| %>
 <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-<%= this_work.pseuds.map{|p| p.byline}.to_sentence.html_safe %> posted a <% if creation.is_a?(Work) then %><%= this_work.backdate ? "backdated" : "new" %> work:<% else %>new chapter of <%= this_work.title %> (<%= this_work.word_count %> words):<% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %><% end %>
-<%= creation.is_a?(Work) ? work_url(this_work) : work_chapter_url(this_work, creation) %>
-
-<%= creation.is_a?(Work) ? this_work.title.html_safe : chapter_title %> (<%= creation.word_count%> words)<% unless @seen[this_work.id] %>
-by <%= this_work.pseuds.map{|p| text_pseud(p)}.to_sentence.html_safe %><% end %>
+<% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+<% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+<% title = this_work.title %>
+<% if creation.is_a?(Work) then
+     link  = work_url(this_work)
+     word_count = this_work.word_count
+   else
+     link  = work_chapter_url(this_work, creation)
+     chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
+   end %>
+<% if creation.is_a?(Work) %>
+<%= if this_work.backdate then
```

Comment:
`then` should be removed, as should the extra space between `word_count` and `)` on line 18. This might also be better in a helper or broken up like:

```
<%= 
  if this_work.backdate
    t('user_mailer.batch_subscription_notification.text.work_backdated', byline: byline, title: title, link: link, word_count: word_count)
  else
    t('user_mailer.batch_subscription_notification.text.work_new', byline: byline, title: title, link: link, word_count: word_count)
  end
%>
```